test/build-combination/Common/main.c

```diff
@@ -314,3 +317,11 @@ void vApplicationPingReplyHook( ePingReplyStatus_t eStatus,
 {
     /* Provide a stub for this function. */
 }
+
+#if ( ipconfigUSE_IPv6 != 0 ) && ( ipconfigUSE_DHCPv6 != 0 )
+    /* DHCPv6 needs a time-stamp, seconds after 1970. */
+    uint32_t ulApplicationTimeHook( void )
```

Comment:
This is good for now, for testing. Actually we need to have an RTC or ask NTP for the current time.

Reply:
Yeah kept for only TCP lib compilation for now.